ksql-engine/src/test/java/io/confluent/ksql/codegen/CodeGenRunnerTest.java

```diff
@@ -42,15 +51,68 @@
     private CodeGenRunner codeGenRunner;
     private FunctionRegistry functionRegistry;
 
+    final private static int COL0_INDEX = 0;
```

Comment:
Could we have these named things like `INT32_FIELD`? As this removes the need to keep scrolling up to work out what _type_ a certain column is, i.e. it makes the code more readable!